src/components/AbstractPageDefinitions.js

```diff
@@ -92,9 +94,22 @@ export default class AbstractPageDefinitions extends Component {
     this.props.history.push(url)
   }
 
+  /**
+   * Opens a Modal that shows the Full Detail View
+   *
+   */
   onInspect(component) {
-    const url = `${ROUTE_INSPECT}/${component.toPath()}`
-    this.props.history.push(url)
+    //const url = `${ROUTE_INSPECT}/${component.toPath()}`
+    //this.props.history.push(url)
```

Comment:
Are we use the URL is not change when the modal is shown?

Reply:
We don't need to use a specific URL for the Modal View since is not deep linked and directly reachable by the browser.
So we can simply use a state property

Reply:
Yes, but let's say I open the Modal and then refresh the browser, if the URL has been changed with the Web history API I'll be able to see the Full Details page.
In addition, changing the URL will allow me to copy and paste this URL to share it.

I guess this is more a product decision. Happy to raise and do it later if necessary

Reply:
That's the purpose of the detail view with deep linking